player/models.py

```diff
@@ -43,7 +43,7 @@ class Room(models.Model):
     }
 
     def __str__(self):
-        return self.name
+        return "{} \n playing: {}".format(self.name, self.current_music)
```

Comment:
Wrong, str.format() should be prefered since "%s" may will be removed !

https://docs.python.org/3.2/tutorial/inputoutput.html#old-string-formatting
"Since str.format() is quite new, a lot of Python code still uses the % operator. However, because this old style of formatting will eventually be removed from the language, str.format() should generally be used."

And second argument, my python teacher rekted me hard for not using format() x)